### ISSUE TITLE
Time units are stripped when value is 0

### ISSUE THREAD

**Issue by herrethan** [less/less.js#2686]
I'm using Less 1.7.5.

Writing `animation-delay: 0s, 1s;` properly outputs this: `animation-delay: 0s, 1s`.

However this:
```
@my-duration: 1s;
animation-delay: 0s, @my-duration;
```

improperly outputs:
```
animation-delay:0,1s
```

It seems the `s` or `ms` unit is stripped from time values when the value is 0, and when it is on the same line as a referenced less variable. 0 without a unit is a invalid syntax for any CSS animation duration.

**Comment by seven-phases-max** [comment]
Don't use `-x` option, it's deprecated since v2.2.0. Use `--clean-css` instead.

**Comment by seven-phases-max** [comment]
Closing as not applicable to current Less versions.

**Comment by lukeapage** [comment]
we also fixed this in 1.5.0.. I just started trying to reproduce..  we do have a regression test for it,  so it may be your minifier and not less?

**Comment by seven-phases-max** [comment]
Hmm, indeed, I did not think it's actually still working properly in `2.5.1` (`s` is not removed by `-x`), can't say for `1.7.5`.
